9-regular-expressions/09-regexp-quantifiers/2-find-html-colors-6hex/task.md

```diff
@@ -1,15 +1,15 @@
-# Regexp for HTML colors
+# Regex para cores hexadecimais
```

Comment:
We have the same point we discussed before about translating the `regexp` to `expressão regular` or `expressões regulares`.

I know that it still a habit to be formed, but we are here to help.

Reply:
I believe that the term `expressão regular` is understandable to the reader at an early stage in their career. But, I consider keeping some English words out of the translation as it keeps the real context of the word, as the term regex is commonly known even in other languages better than the translation itself. Particularly, I consider the same premise as reserved words in javascript, such as 'const'. (it is not relevant to translate to `constante`)